test/spec-urls.test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+const assert = require('assert');
+const specData = require('browser-specs');
+const { walk } = require('../utils');
+
+describe('spec_url data', () => {
+  it('spec_urls only use allow listed hosts by w3c/browser-specs (and our exception list)', () => {
+    const specURLs = [];
+
+    for (const { compat } of walk()) {
+      const { spec_url } = compat;
+      const specs = [].concat(spec_url || []); // coerce spec_url to array, or empty array if undefined
+      specURLs.push(...specs);
+    }
+
+    const specsFromBrowserSpecs = [
+      ...specData.map(spec => spec.url),
+      ...specData.map(spec => spec.nightly.url),
+      ...specData.map(spec => spec.series.nightlyUrl),
+    ];
+
+    const specsExceptions = [
+      'https://wicg.github.io/controls-list/',
+      'https://w3c.github.io/setImmediate/',
```

Comment:
Should be removed if/when https://github.com/mdn/browser-compat-data/pull/10746 lands